internal/operators/ocs/ocs_operator.go

```diff
@@ -92,6 +94,43 @@ func (o *operator) GetMonitoredOperator() *models.MonitoredOperator {
 }
 
 // GetHostRequirements provides operator's requirements towards the host
-func (o *operator) GetHostRequirements(context.Context, *common.Cluster, *models.Host) (*models.ClusterHostRequirementsDetails, error) {
-	return &models.ClusterHostRequirementsDetails{}, nil
+func (o *operator) GetHostRequirements(_ context.Context, cluster *common.Cluster, host *models.Host) (*models.ClusterHostRequirementsDetails, error) {
+	numOfHosts := len(cluster.Hosts)
+
+	if numOfHosts < 3 {
+		return nil, fmt.Errorf("number of hosts is less than 3")
```

Comment:
Validation regarding the host count are done elsewhere. Number of masters will not be always 3 (there is SNO mode). So I think we should not check the host number here cc: @gamli75

Reply:
@slaviered , OCS won't work for SNO mode. , i.e, OCS requires atleast 3 hosts.